### PR TITLE
Update rx config note

### PR DESCRIPTION
**Pull request by ctzsnooze**
Re-opening PR #2562 with some minor text modifications.

This provides more comprehensive help than the previous tooltip.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
@ctzsnooze this needs a rebase

**Review by limonspb — DISMISSED** [review]

File: locales/en/messages.json [hunk 1]
```diff
@@ -1993,6 +1990,9 @@
...
... etc).  Choose the correct ‘Channel Map’ for your radio.  Bind the receiver and check that the Roll, Pitch and other channel bar graphs move correctly with stick movement.  You may need to reconfigure Map, Mode, Channels or Mix so that, for example, moving the Roll stick changes Roll values.  Adjust the channel endpoint or range values in the transmitter so that all channels go from ~1000 to ~2000.  Trim the midpoint to 1500.  RC Smoothing is best left at defaults.  For more information, please read receiver chapter of the <strong><a href=\"https://github.com/betaflight/betaflight/blob/master/docs/Rx.md\" target=\"_blank\" rel=\"noopener noreferrer\">documentation</a></strong>.<br /></br /><span class=\"message-negative\"><strong>Important</strong></span>:<br /> &#8226; Don't forget to configure and check failsafe in the Failsafe tab!<br /> &#8226; It is highly recommended to use the latest Tx firmware.  OpenTx or EdgeTx users should disable the hardware> ADC filter with Betaflight.  For more information see <strong><a href=\"https://github.com/betaflight/betaflight/blob/master/docs/Rx.md#disable-adc-filter\" target=\"_blank\" rel=\"noopener noreferrer\">documentation</a></strong>"
```
Comment (Asizon) [inline locales/en/messages.json:1994]: disable adc filter url is wrong, should be https://github.com/betaflight/betaflight/blob/master/docs/Rx.md#disabling-the-opentxedgetx-adc-filter

**Review by asizon — CHANGES_REQUESTED** [review]

**Review by asizon — DISMISSED** [review]

**Review by limonspb — DISMISSED** [review]

**Review by haslinghuis — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Tried to bring master in, but that didn't solve it. Please rebase manual

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
Fixed the link syntax, but closing in favour of #2671 which is intended to give a clean face for the receiver tab for 4.3

**Pull request closed without merging**